pkg/server/config.go

```diff
@@ -0,0 +1,47 @@
+package server
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/viper"
+)
+
+// Environment Vars
+const (
+	EnvEnvironment    = "ENVIRONMENT"
+	EnvConfigLocation = "CONFIG_PATH"
+)
+
+// Config exposes Server configuration options
+type Config struct {
+	BasePath       string
+	MetricsEnabled bool
+	Name           string
+	Port           int
+}
+
+// NewConfig constructs a new *server.Config instance with defaults
+func NewConfig() *Config {
+	viper.SetConfigName(getEnvironmentVariable(EnvEnvironment, "review"))
+	viper.AddConfigPath(getEnvironmentVariable(EnvConfigLocation, "./config"))
+
+	err := viper.ReadInConfig()
+	if err != nil {
+		panic(fmt.Errorf("Fatal error config file: %s", err))
+	}
+
+	return &Config{
+		BasePath:       "/api/v1",
+		Name:           viper.GetString("host"),
+		Port:           viper.GetInt("port"),
+		MetricsEnabled: viper.GetBool("metrics"),
```

Comment:
I have an idea for based on this flag we'd enable a default Prometheus handler

Reply:
I'll make a trello ticket to handle this! https://trello.com/c/5bGJUUOZ/41-handle-passing-flags-at-runtime-to-enable-things-like-metrics